content/images/services-appdev.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Services-AppDev",
```

Comment:
You don't have to name Image content records in Pascal-Kebab-Case; you can just name them in regular title case, and Netlify CMS will save them as lower-kebab-case for use as slug references.